astropy/tests/__init__.py

```diff
@@ -4,8 +4,3 @@
 for writing tests, and general tests that are not associated with a
 particular package.
 """
-from __future__ import (absolute_import, division, print_function,
-                        unicode_literals)
-
-from . import helper
-from .disable_internet import turn_off_internet, turn_on_internet
```

Comment:
Should we consider this as being part of the public API and therefore keep the last import?